src/tpm2.c

```diff
@@ -187,9 +188,16 @@ static int TPM2_CommandProcess(TPM2_CTX* ctx, TPM2_Packet* packet,
             }
 
         #ifndef WOLFTPM2_NO_WOLFCRYPT
-            rc =  TPM2_GetName(ctx, info->inHandleCnt, 0, &name1);
-            rc |= TPM2_GetName(ctx, info->inHandleCnt, 1, &name2);
-            rc |= TPM2_GetName(ctx, info->inHandleCnt, 2, &name3);
+            UINT32 handleValue;
```

Comment:
Please move variable declaration to top of function or put within a brace.

Reply:
This is still an issue. Variables must be declared at top of function or brace.

Reply:
Indeed. Fixed